image_tools/src/cam2image.cpp

```diff
@@ -125,7 +125,11 @@ int main(int argc, char * argv[])
         convert_frame_to_message(flipped_frame, i, msg);
       }
       if (show_camera) {
-        cv::imshow("cam2image", frame);
+        // NOTE(esteve): Use C version of cvShowImage to avoid this on Windows:
+        // http://stackoverflow.com/questions/20854682/opencv-multiple-unwanted-window-with-garbage-name
+        CvMat cvframe;
```

Comment:
It's cleaner to use http://docs.opencv.org/modules/core/doc/basic_structures.html#mat-operator-cvmat